src/openrct2/paint/Paint.cpp

```diff
@@ -549,19 +550,20 @@ static void PaintAttachedPS(rct_drawpixelinfo* dpi, paint_struct* ps, uint32_t v
     {
         auto screenCoords = ScreenCoordsXY{ attached_ps->x + ps->x, attached_ps->y + ps->y };
 
-        uint32_t imageId = PaintPSColourifyImage(attached_ps->image_id, ps->sprite_type, viewFlags);
+        auto imageId = PaintPSColourifyImage(
+            ImageId::FromUInt32(attached_ps->image_id, attached_ps->tertiary_colour), ps->sprite_type, viewFlags);
```

Comment:
It previously used `ps->tertiary_colour` to set the tertiary colour. So shouldn’t this line read like this:

```suggestion
            ImageId::FromUInt32(attached_ps->image_id, ps->tertiary_colour), ps->sprite_type, viewFlags);
```